lib/src/middleware/edit_select_mode_change.dart

```diff
@@ -17,6 +17,7 @@ const selectable_css_style_non_domain_or_end = {
 };
 
 const selectable_css_style_domain = {
+  'filter': 'url("#shadow")',
```

Comment:
Is it possible to do this without turning the domain pink and making it thicker? I was only doing those as a stopgap measure since the shadow filter didn't work.

I tried removing `'stroke': 'hotpink'` and `'stroke-width': '5pt'` (i.e., only add the shadow filter), and for me, the problem came back.

Reply:
![scadnano-domain-shadow-filter-remove-pink-hover](https://user-images.githubusercontent.com/31255679/94053302-0688ca00-fd8f-11ea-86e5-e4bf69435148.gif)
Strange, seems okay for me?